python-sdk/langwatch/src/langwatch/utils/utils.py

```
@@ -0,0 +1,25 @@
+def safe_get(d: Union[Dict[str, Any], BaseModel], *keys: str) -> Optional[Any]:
+    for key in keys:
+        if isinstance(d, dict):
+            d = d.get(key, None)
+        if hasattr(d, key):
+            d = getattr(d, key)
+        else:
+            return None
+    return d
+
+
+def list_get(l, i, default=None):
```

Ruff (0.8.2)

Ambiguous variable name: `l`

(E741)